src/clzones.h

```diff
@@ -440,8 +440,12 @@ class zone_manager
                   const tripoint &start, const tripoint &end,
                   const shared_ptr_fast<zone_options> &options = nullptr, const bool personal = false,
                   bool silent = false, map *pmap = nullptr );
+        // get first matching zone
```

Comment:
It's returning `ret.back();`, so it's actually the last matching zone, is it not?

Reply:
Good catch. Thanks!